src/google/cloud/ndb/model.py

```diff
@@ -523,8 +523,43 @@ def _entity_from_ds_entity(ds_entity, model_class=None):
     entity = model_class()
     if ds_entity.key:
         entity._key = key_module.Key._from_ds_key(ds_entity.key)
+
     for name, value in ds_entity.items():
         prop = getattr(model_class, name, None)
+
+        # Backwards compatibility shim. NDB previously stored structured
+        # properties as sets of dotted name proprties. Datastore now has native
```

Comment:
Typo on "proprties", but the just merged spell checker should catch this before your next push (once you rebase, of course).

Reply:
Actually, it looks like spell checker doesn't find it because comments don't make it into docs.

Reply:
Ha, I thought it was in a doc string. Duh.